### ISSUE TITLE
Are there plans to keep the code in Sync with the original Repo?

### ISSUE THREAD

**Issue by Morphexe** [ryancheung/RecastSharp#4]
Is the code manually generated, or is it being created with a tool ? 
I am interesting in wether this will keep getting updated from the original repo, or if we need to manually sync changes !

Best regards!

**Comment by ryancheung** [comment]
Yeah, code from original cpp repo will be synced periodicly, and it should be easy.
